python/ray/tests/test_placement_group_metrics.py

```diff
@@ -0,0 +1,70 @@
+from collections import defaultdict
```

Comment:
I have started to notice that the e2e metrics tests take quite a while to run (probably due to the latency in exporting and scraping). 
As we add more tests and metrics, should we consider a more efficient and easier way to test the metrics e2e? 

Also cc @scv119

Reply:
There seems to be a big 10s delay before metrics show up, even with setting the interval small. Any idea what this delay is? That's the majority of waiting time.